js/game.js

```diff
@@ -206,92 +187,67 @@ Game.prototype.build = function(xOnBoard,yOnBoard) {
     this.selectedBuilding = undefined;
     View.deselectBuilding();
   }
-}
+};
 
 Game.prototype.buildCurrentBuildOrder = function() {
-  var building = this.currentBuildOrder
-  if (building) {
-    if (building.completed) {
-      building = undefined;
```

Comment:
I don't think this does anything. You're setting the variable to `undefined`, not the model property. The variable is never returned. Just removed the else statement.